features/bevragen/autorisatie-gba.feature

```diff
@@ -0,0 +1,590 @@
+
+  De autorisatie voor gegevens is vastgelegd in Rubrieknummer ad hoc (35.95.60). 
+  Deze bevat een lijst met rubrieknummers (spatie gescheiden) van gegevens die geleverd mogen worden. 
+  Een rubrieknummer is het nummer voor een gegeven zoals gedefinieerd in het Logisch Ontwerp BRP (LO BRP) zonder voorloopnullen voor categorienummer (dus 5 of 6 cijfers).
+  Voor het toepassen van de autorisatie moet het rubrieknummer worden vertaald naar het corresponderende veld in de BRP API resource (of andersom). Deze vertaling staat beschreven in /docs/BRP-LO GBA mapping.md.
+
+  Autorisatie wordt verkregen met behulp van een Oauth 2 token. 
+  In het verkregen token is de afnemerindicatie opgenomen die de afnemer uniek identificeert. Op basis van de afnemerindicatie kan de autorisatie worden opgezocht.
+  Wanneer de afnemer een gemeente is, is er ook een gemeentecode opgenomen in de Oauth token.
+
+  # Autorisaties worden opgeslagen in de tabel lo3_autorisatie. Deze bevat minimaal:
+  #  - autorisatie_id: technische identificatie (primary key) = max(autorisatie_id)+1
+  #  - afnemer_code: vullen met 'Afnemersindicatie (35.95.10)' in Gegeven, wordt ook opgenomen in Oauth token gestuurd bij Als stap
+  #  - tabel_regel_start_datum: vullen met 'Datum ingang (35.99.98)' in Gegeven stap
+  #  - tabel_regel_eind_datum: vullen met 'Datum beëindiging tabelregel (35.99.99)' in Gegeven stap
+  #  - ad_hoc_medium: vullen met 'Medium ad hoc (35.95.67)' in Gegeven stap, vaste waarde 'N' (betekent dat de autorisatie er wel is)
+  #  - ad_hoc_rubrieken: vullen met waarde voor 'Rubrieknummer ad hoc (35.95.60)' in Gegeven stap
+
+  Voor één afnemer kunnen er meerdere rijen zijn in de autorisatietabel, maar daarvan kan er maar één actueel zijn. Alleen de actuele mag worden gebruikt.
+  Een autorisatie is actueel wanneer de Datum ingang (35.99.98) in het verleden ligt en Datum beëindiging tabelregel (35.99.99) leeg is of in de toekomst ligt.
+
+
+  # To Do (t.z.t.): alleen regel gebruiken met datum ingang <= vandaag
+
+  # To Do (t.z.t.): regels voor leveren persoonsgegevens met geheimhouding aan derde (35.95.12)
+
+  # To Do (t.z.t.): regels voor voorwaarderegel ad hoc (35.95.61)
+
+
+  Rule: Een afnemer is geautoriseerd voor ad hoc gegevensverstrekking als Medium ad hoc (35.95.67) de waarde “N” of “A” heeft
+    Wanneer Medium ad hoc (35.95.67) leeg is of een andere waarde dan "N" of "A" heeft, wordt een foutmelding gegeven
+
+    @fout-case
+    Scenario: Afnemer is niet geautoriseerd voor ad hoc gegevensverstrekking
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 010120 010210                   |                          | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
```

Comment:
Moet 'afnemerID' en 'gemeenteCode' niet opgenomen worden in de 'columnNameMap' van 'step_defs.js'?

Reply:
In deze given context zijn het geen kolommen van een database tabel.
Rubrieknummer ad hoc (35.95.60), Medium ad hoc (35.95.67) en Datum ingang (35.99.98) zijn dat wel en die zijn al toegevoegd aan de step_defs.js (develop)

Reply:
Ok, duidelijk.